config.toml

```
@@ -203,6 +203,13 @@ time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
 
+[languages.ru]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
```

Something like:
```suggestion
description = "Контейнерная оркестровка, готовы на производст"
```
maybe?

---

I was thinking the same but the description is the same for all languages.

---

Actually, some localizations do translate this. Check out https://github.com/kubernetes/website/pull/16003/files#diff-618063036395fe9ee107f22b46c9eadeR245